main.go

```diff
@@ -1,6 +1,13 @@
 package main
 
+import "flag"
+
+var cutefood = true
```

Comment:
As you mentioned in you comment we really don't want such global variable. What I recommend you to do is passing the flag to the `NewGame()` which can pass it to the `Arena` where it will make use of it to choose which food to use